src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -191,6 +191,8 @@ export const addCustomElementInputs = (
   const components = buildCtx.components;
   // an array to store the imports of these modules that we're going to add to our entry chunk
   const indexImports: string[] = [];
+  const indexExports: string[] = [];
```

Comment:
might be helpful to have a short comment above this line (and above the line below) explaining what these arrays are for?